File: Dinner/Python/strong_number.py
numberString = "165"
number = int(numberString)

count = 0
total = 0
for element in numberString:
	digit = int(numberString[count])
	factorial = 1
	print(digit)

	for element in range (1, digit + 1):	
		factorial *= element
	total += factorial
	count +=1
if (number == total):
	print(f"{number} is a Strong Number")
else:	
	print(f"{number} is not a  Strong Number")

File: Dinner/Java/StrongNumber.java
public class  StrongNumber{
	public static void main(String... args){
		int number = 15;
		int  numberCheck = number;
		int total = 0;
		for(int count = 0; numberCheck > 0; count++){
			int  digit = numberCheck % 10;
				int factorial = 1;
			for(int counter = 1; counter <= digit; counter++){
				factorial *= counter;
			}
				total += factorial;
				numberCheck = numberCheck / 10; 
		}
			if (number == total)
				System.out.println(number + " is a StrongNumber");
			else
				System.out.println(number + " is not StrongNumber");


	}


}